libs/logic-apps-shared/src/utils/src/lib/models/uiInteractionData.ts

```diff
@@ -0,0 +1,10 @@
+import type { DiscoveryOpArray } from './operationDiscoveryResults';
+
+export interface UiInteractionData {
+  graphId?: string;
+  parentId?: string;
+  childId?: string;
+  nodeId?: string;
+  preloadedOperations?: DiscoveryOpArray;
```

Comment:
This looks very specific to the use case that Power Automate has for this menu and likely is not that useful to other consumers. What is the reason to pass the object here rather than determine it in the (PA) host?

Reply:
The search service wasn't working properly before. Now I figured out how the search service should work, then I think I can remove this.